packages/terra-button/src/terra-dev-site/test/button/ActiveVariantButtons.test.jsx

```diff
@@ -1,61 +1,63 @@
 import React from 'react';
 import IconSquare from 'terra-icon/lib/icon/IconSquareSymbol';
 import classNames from 'classnames/bind';
+import buttonStyles from 'terra-button/lib/Button.module.scss';
 import Button from '../../../Button';
 import styles from './ButtonTestCommon.module.scss';
 
 const cx = classNames.bind(styles);
+const cy = classNames.bind(buttonStyles);
 
 const NeutralButton = () => (
   <div id="neutral" className={cx('spacing')}>
-    <Button id="neutralActive" text="Neutral-Active" variant="neutral" className="is-active" />
+    <Button id="neutralActive" text="Neutral-Active" variant="neutral" className={cy('is-active')} />
```

Comment:
Fixed the class names binding loophole, broke the tests

![image](https://user-images.githubusercontent.com/6720991/79263094-e8ba4f00-7e57-11ea-94a2-36974b632de3.png)

Reply:
For a temporary workaround to move this PR forward I think this makes sense, in the long run we might consider flagging this to update later. Either find a way to press the active state in a test or find another way to test the functionality. I don't think we should rely on local class names outside of the class for testing.